pages/dr5/issues.rst

```diff
@@ -44,3 +44,18 @@ CCDs covering regions like CCD N26 in the figure would be dropped from DR5.
    :height: 500
    :width: 700
    :scale: 85
+
+Update to ``ccds-annotated-`` file in early December, 2017
+=====================================================================
```

Comment:
This is just a minor quibble, but the underlining is supposed to be the same length as the title text.